fastchat/conversation.py

```diff
@@ -118,6 +118,14 @@ def get_prompt(self) -> str:
                 else:
                     ret += role + ": " + "<s>"
             return ret
+        elif self.sep_style == SeparatorStyle.ROBIN:
```

Comment:
Can it be covered with an existing separator style?

Reply:
Hi. After checking other existing separator styles, I can't find the suitable one. So I think it can not be covered by others.